bip-0360.mediawiki

```diff
@@ -0,0 +1,596 @@
+<pre>
+  BIP: 360
+  Title: QuBit: SegWit v3 spending rules (P2QRH)
+  Layer: Consensus (soft fork)
+  Author: Hunter Beast <hunter@surmount.systems>
+  Comments-Summary: No comments yet.
+  Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-0360
+  Status: Draft
+  Type: Standards Track
+  Created: 2024-12-18
+  License: BSD-3-Clause
+</pre>
+
+
+== Introduction ==
+
+=== Abstract ===
+
+This document proposes the introduction of a new output type using signatures based on Post-Quantum Cryptography (PQC).
+This approach for adding a post-quantum secure output type does not require a hard fork or block size increase.
+
+=== Copyright ===
+
+This document is licensed under the 3-clause BSD license.
+
+=== Motivation ===
+
+The primary threat to Bitcoin from Cryptoanalytically-Relevant Quantum Computers (CRQCs) is their potential to break
+the cryptographic assumptions of Elliptic Curve Cryptography (ECC), which secures Bitcoin's signatures and Taproot
+commitments. Specifically, [https://arxiv.org/pdf/quant-ph/0301141 Shor's algorithm] enables a CRQC to solve the
+Discrete Logarithm Problem (DLP) exponentially faster than classical methods<ref name="shor">Shor's algorithm is
+believed to need 10^8 operations to break a 256-bit elliptic curve public key.</ref>, allowing the derivation of private
+keys from public keys—a process referred to as quantum key decryption. Importantly, simply increasing the public key
+length (e.g., using a hypothetical secp512k1 curve) would only make deriving the private key twice as hard, offering
+insufficient protection. The computational complexity of this attack is further explored in
+[https://pubs.aip.org/avs/aqs/article/4/1/013801/2835275/The-impact-of-hardware-specifications-on-reaching ''The impact
+of hardware specifications on reaching quantum advantage in the fault-tolerant regime''].
+
+This proposal aims to mitigate these risks by introducing a Pay to Quantum Resistant Hash (P2QRH) address type that
```

Comment:
This proposal defines a new _output_ type, not address type. An address is merely an encoding of an output script, and in this case an existing address type (Bech32m) is used.

```suggestion
This proposal aims to mitigate these risks by introducing a Pay to Quantum Resistant Hash (P2QRH) output type that
```